acpi.adoc

```diff
@@ -38,10 +38,6 @@ software-defined interfaces and the complexity in supporting ACPI is reduced.
 The following additional BRS requirements apply:
 
 * No FACS table.
-* Both interrupt-signaled and GPIO-signaled ACPI Events are supported
```

Comment:
This is actually not referenced in the spec. There's a bug open for that, but interrupt-signaled events are not documented as a choice for hw-reduced systems.

Reply:
ACPI Specification, Relase 6.5, p 290:

> Hardware-Reduced ACPI platforms use GPIO-signaled ACPI Events, or Interrupt-signaled ACPI events. Note that
any ACPI platform may utilize GPIO-signaled and/or Interrupts-signaled ACPI events (in other words, these events
are not limited to Hardware-reduced ACPIvplatforms).

Reply:
So this sounds like a reference to the section is useful.

Reply:
Btw, I had filed https://mantis.uefi.org/mantis/view.php?id=2377 to "fix" this in the ACPI spec and no one in ASWG provided feedback that 5.6 ACPI Event Programming Model was a section that already covered this.

You could say this proves something about the length/organization of the ACPI spec, sure, but that's not relevant to our activities on the BRS. As far as the BRS goes, it shows the importance of guiding text pointing out ACPI sections to focus on. Does that go in the normative section? It doesn't (since it *exists* in a dependent spec), but it sure should go into the guidance (non-normative) portion of the BRS document.